examples/datasources/ec2_client_vpn_export_client_config_json/data-source.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_providers {
+    awsutils = {
+      source = "cloudposse/awsutils"
+    }
```

Comment:
please add this comment https://github.com/cloudposse/terraform-provider-utils/blob/main/examples/data-sources/utils_spacelift_stack_config/data-source.tf#L5

and this Makefile https://github.com/cloudposse/terraform-provider-utils/blob/main/Makefile

so the development and testing could be done locally (otherwise, the provider version that is already in the Terraform registry will be tested)

Reply:
There's already a similar `makefile` in the root. Do you want a new one in examples?

Reply:
no, one in the root is ok @r351574nc3